posthog/queries/trends/breakdown.py

```diff
@@ -403,7 +403,11 @@ def _get_breakdown_value(self, breakdown: str) -> str:
                 breakdown_value, _ = get_property_string_expr("events", breakdown, "%(key)s", "properties")
 
         if self.filter.using_histogram:
-            return f"toFloat64OrNull(toString({breakdown_value}))"
+            breakdown_value = f"toFloat64OrNull(toString({breakdown_value}))"
+
+        if self.filter.breakdown_normalize_url:
+            breakdown_value = f"if( length(trim(TRAILING '/?#' from {breakdown_value})) = 0, '/', trim(TRAILING '/?#' from {breakdown_value}))"
```

Comment:
Clickhouse also has an `empty()` function, might be a bit more readable. Also I suppose, we could do the `trim(TRAILING)` operation once if it was defined as an extra column (`... AS raw_normalized_breakdown_value, if(...)`)

Reply:
Ah yeah, I didn't take a good look at the complexity of that. It should be fine as is too.